src/utils/updateContainerApp/updateContainerApp.ts

```diff
@@ -7,7 +7,7 @@ import { ContainerApp, ContainerAppsAPIClient } from "@azure/arm-appcontainers";
 import { getResourceGroupFromId } from "@microsoft/vscode-azext-azureutils";
 import { IActionContext, createSubscriptionContext, nonNullProp } from "@microsoft/vscode-azext-utils";
 import { AzureSubscription } from "@microsoft/vscode-azureresources-api";
-import { createContainerAppsAPIClient } from "./azureClients";
+import { createContainerAppsAPIClient } from "../azureClients";
 
 export async function updateContainerApp(context: IActionContext, subscription: AzureSubscription, containerApp: ContainerApp, updatedSetting?: Omit<ContainerApp, 'location'>): Promise<void> {
```

Comment:
I know that this isn't technically a command, but it still seems strange that this is in the utils folder with other files in command like pattern. I would prefer we just move it somewhere with the commands since everything that is using it is in commands anyway.